spec/models/content_page_spec.rb

```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe ContentPage, type: :model do
   let(:expected_exception_message) do
-    /should only contain alphabetic, numeric, comma, fullstop, hyphen and space characters/
+    /#{ContentPage::TITLE_FORMAT_ERROR_MESSAGE}/
   end
```

Comment:
I generally prefer the test not rely on the implementation, so I would have just stayed with the explicit expectation.

Reply:
Got you